core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -302,9 +302,6 @@ class ReplicaManager(val config: KafkaConfig,
     new DelayedOperationPurgatory[DelayedDeleteRecords](
       "DeleteRecords", config.brokerId,
       config.deleteRecordsPurgatoryPurgeIntervalRequests))
-  val delayedElectLeaderPurgatory = delayedElectLeaderPurgatoryParam.getOrElse(
```

Comment:
please remove `delayedElectLeaderPurgatoryParam` also

Reply:
Also, this also remove the following metrics.
1. kafka.server:type=DelayedOperationPurgatory,delayedOperation=ElectLeader,name=PurgatorySize
1. kafka.server:type=DelayedOperationPurgatory,delayedOperation=ElectLeader,name=NumDelayedOperations

please add them into `zk2kraft.html`

Reply:
Interesting, we still allow elect leader to be called. Why are there metrics no longer relevant?

Reply:
The zk broker waits for the metadata update before returning a response to the `ElectLeader` request, whereas the Kraft broker does not. Consequently, the Kraft broker does not need delayedOperation metrics for ElectLeader requests. Instead, kraft broker has `ForwardingManager` metrics to monitor the forwarded requests.

Reply:
Thanks for the explanation.